developer-workflow/lang-changes.rst

```diff
@@ -41,25 +47,104 @@ Council can truly classify something as Pythonic, you can read the
 Suggesting new features and language changes
 --------------------------------------------
 
-The `Ideas Discourse category`_
-is specifically intended for discussion of new features and language changes.
-Please don't be disappointed if your idea isn't met with universal approval:
-as the :pep:`long list of Withdrawn and Rejected PEPs
-<0#rejected-superseded-and-withdrawn-peps>`
-in the :pep:`PEP Index <0>` attests,
-and as befits a reasonably mature programming language,
-getting significant changes into Python isn't a simple task.
+Proposing a change involves the following steps:
+
+- Describe your idea in detail.
+
+- Engage in discussion about the idea.
+
+- Rarely, the idea will advance to a formal proposal stage.
+
+
+Describe your idea
+^^^^^^^^^^^^^^^^^^
+
+The `Ideas category in Discourse <ideas_>`_ is specifically intended for discussion
+of new features and language changes. Make your proposal as a `new topic
+<ideas_>`_ there.
+
+Your proposal needs to be **detailed**. Describe the change you want to make,
+why you want to make it, and what benefits it will bring to Python users. Be
+specific. Show that you have considered the effect of the change on existing
+code.
+
+Some things you should **not** do:
+
+- **Don't** format your idea as a Python Enhancement Proposal (*PEP*).
+  It's good to use the PEP template as a guide for what information to include,
+  but don't try to write a full PEP until the idea has been accepted for
+  further consideration.
+
+- **Don't** create a list of proposed changes. Each idea needs its own
+  discussion. Proposing a change is significant work. You will need to spend
+  time on each proposal. A tossed-off list will not be taken seriously.
+
+When justifying your idea, these are not good reasons to make a change:
```

Comment:
```suggestion
When justifying your idea, these are **not good reasons** to make a change:
```
To highlight the negateion as in the section before.

Reply:
Done.